README.md

```diff
@@ -58,50 +58,33 @@ To get the source code from GitHub, type
 
 > **Note**
 > 
-> The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [PEAR package manager](http://pear.php.net/manual/en/installation.php).
+> The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [Composer package manager](http://getcomposer.org).
 
 
 ##Install via Composer
 
-1. Create a file named **composer.json** in the root of your project and add the following code to it:
+* Create a file named **composer.json** in the root of your project and add the following code to it:
 ```json
     {
-      "require": {
-        "microsoft/windowsazure": "*"
-      },      
-      "repositories": [
-        {
-          "type": "pear",
-          "url": "http://pear.php.net"
-        }
-      ],
-      "minimum-stability": "dev"
+        "_comment": "php >= 5.5 is ONLY needed when using Media Services API",    
+        "require": {        
+            "php": ">=5.5",
+            "microsoft/windowsazure": "*"
```

Comment:
Since the library tracks the dev version for [`0.4` in `branch-alias`](https://github.com/Azure/azure-sdk-for-php/blob/master/composer.json#L32), `^0.4@dev` only will match that version, and not any previous tagged release such as `0.4.0` or `0.4.1` because the stability flag which is present in the constraint (`@dev`).